exercises/sum.cpp

```diff
@@ -6,3 +6,18 @@
   Insert the second number: 2
   Sum: 3
 */
+
+#include <stdio.h>
+
+int x, y;
+
+int main(){
+	printf("Insert the first number: ");
+	scanf("%d", &x);
+
+	printf("Insert the second number: ");
+	scanf("%d", &y);
+
+	printf("Sum: %d", x + y);
+	return 0;
```

Comment:
stai usando C non C++ :(

potresti sostituire 0 con EXIT_SUCCESS in modo da migliorare la leggibilità del codice
```c++
#define EXIT_SUCCESS 0
...
return EXIT_SUCCESS;
```